cura/Arranging/ArrangeObjectsJob.py

```diff
@@ -8,17 +8,20 @@
 from UM.Message import Message
 from UM.Scene.SceneNode import SceneNode
 from UM.i18n import i18nCatalog
+from cura.Arranging.GridArrange import GridArrange
 from cura.Arranging.Nest2DArrange import arrange
 
 i18n_catalog = i18nCatalog("cura")
 
 
 class ArrangeObjectsJob(Job):
-    def __init__(self, nodes: List[SceneNode], fixed_nodes: List[SceneNode], min_offset = 8) -> None:
+    def __init__(self, nodes: List[SceneNode], fixed_nodes: List[SceneNode], min_offset = 8,
+                 grid_arrange: bool = False) -> None:
```

Comment:
I'm not a fan of this as this is a "bool trap". This can be prevented by doing the following:
```suggestion
    def __init__(self, nodes: List[SceneNode], fixed_nodes: List[SceneNode], min_offset = 8,
                 *, grid_arrange: bool = False) -> None:
```
This means that you can only use the grid_arrange as a kwarg and not as an unnamed one. 

So `ArrangeObjectsJob([], [], 8, grid_arrange = True)` is allowed, whereas `ArrangeObjectsJob([], [], 8, True)` is not

Reply:
fixed in https://github.com/Ultimaker/Cura/pull/15496/commits/118f49a0520bdd1a3983bbf33b49c3ed8a645fb1